molo/core/tests/test_views.py

```diff
@@ -111,3 +111,9 @@ def test_issue_with_django_admin_not_loading(self):
 
         response = self.client.get(reverse('admin:index'))
         self.assertEquals(response.status_code, 200)
+
+    def test_versions_comparison(self):
+        response = self.client.get(reverse('versions'))
+        self.assertContains(
```

Comment:
I think you need to mock the different modules and versions and also test the version numbers and git compare urls.  I'll also do some digging around to find examples for you